data/widgets/sprite_properties.xml

```diff
@@ -43,6 +43,16 @@
         </hbox>
       </hbox>
     </grid>
+
+    <grid columns="3">
+      <separator text="@.tilesets" horizontal="true" cell_hspan="3" />
+
+      <hbox></hbox>
+      <view expansive="true" cell_align="horizontal" cell_hspan="2" minheight="63">
+        <listbox id="tilesets"></listbox>
+      </view>
+    </grid>

```

Comment:
I think this could be just a `<vbox>` (added `id="tilesets_placeholder"` as we could do a `tilesetsPlaceholder()->setVisible(false)` if the sprite doesn't have tilesets):
```suggestion
    <vbox expansive="true" id="tilesets_placeholder">
      <separator text="@.tilesets" horizontal="true" />
      <view expansive="true" minheight="63">
        <listbox id="tilesets"></listbox>
      </view>
    </vbox>
```
In this way the listbox expands correctly if the window is bigger:
![image](https://github.com/aseprite/aseprite/assets/39654/98b487aa-3c75-4b79-ae72-1899556b0b12)